### PR TITLE
Add excel fit to width flag

### PR DESCRIPTION
**Pull request by codeMonkeysBe**

### CODE REVIEW

File: OfficeToPDF/ExcelConverter.cs [hunk 1]
```diff
@@ -410,6 +408,14 @@ public static new int Convert(String inputFile, String outputFile, Hashtable opt
 
                 workbook.SaveAs(tmpFile, fmt, Type.Missing, Type.Missing, Type.Missing, false, XlSaveAsAccessMode.xlNoChange, Type.Missing, false, Type.Missing, Type.Missing, Type.Missing);
 
+                // Fit to to one page wide.
+                if (fitToPageWidth)
+                {
+                    templatePageSetup["FitToPagesWide"] = 1;
+                    templatePageSetup["FitToPagesTall"] = false;
```
Comment (xonstone) [inline OfficeToPDF/ExcelConverter.cs:415]: Q: shouldn't this be a seperate flag?

Comment (codeMonkeysBe) [reply OfficeToPDF/ExcelConverter.cs:415]: In zoom convo we decided that we would have only 1 flag for the entire functionality.  Still open to change this though.

Comment (codeMonkeysBe) [reply OfficeToPDF/ExcelConverter.cs:415]: Decided to keep this as one logical flag.

Comment (becoded) [inline OfficeToPDF/ExcelConverter.cs:415]: Q: don't we also need landscape instead of portrait mode?

Comment (codeMonkeysBe) [reply OfficeToPDF/ExcelConverter.cs:415]: Right now we don't force landscape or portrait, it just assumes the user's setting. ( landscape is the default ).

I could add the option, but not sure if we want to make that choice for the user.

Comment (becoded) [reply OfficeToPDF/ExcelConverter.cs:415]: I added it in the consequences doc so support can pick it up in their documentation

**Pull request merged**